src/handlers/handlerUtils.ts

```diff
@@ -951,6 +954,10 @@ export function constructConfigFromRequestHeaders(
     awsRegion: requestHeaders[`x-${POWERED_BY}-aws-region`],
   };
 
+  const workersAiConfig = {
+    accountId: requestHeaders[`x-${POWERED_BY}-workers-ai-account-id`],
```

Comment:
Should we change this to workersAIAccountId instead of just accountId to avoid confusion in future. The idea here is to have the same naming for header as well as the config field.

For example: bedrockConfig is
```
const bedrockConfig = {
    awsAccessKeyId: requestHeaders[`x-${POWERED_BY}-aws-access-key-id`],
    awsSecretAccessKey: requestHeaders[`x-${POWERED_BY}-aws-secret-access-key`],
    awsSessionToken: requestHeaders[`x-${POWERED_BY}-aws-session-token`],
    awsRegion: requestHeaders[`x-${POWERED_BY}-aws-region`],
  };
```

Azure is an exception because it was written way back. But now that we have good number of providers, its better to follow this convention.

Reply:
@VisargD good point, fixed in https://github.com/Portkey-AI/gateway/pull/269/commits/4a95d4ff95eceb364ed639b13c594147864ca363